srcpkgs/gparted/template

```diff
@@ -1,18 +1,23 @@
 # Template file for 'gparted'
 pkgname=gparted
-version=1.1.0
+version=1.2.0
 revision=1
 build_style=gnu-configure
 configure_args="--enable-libparted-dmraid"
 hostmakedepends="intltool itstool pkg-config"
 makedepends="libglib-devel gtkmm-devel libparted-devel yelp-tools"
 depends="desktop-file-utils hicolor-icon-theme"
+checkdepends="xvfb-run"
 short_desc="Gnome Partition Editor"
 maintainer="teldra <teldra@rotce.de>"
 license="GPL-2.0-or-later"
 homepage="https://gparted.sourceforge.io"
 distfiles="${SOURCEFORGE_SITE}/gparted/gparted/gparted-${version}/gparted-${version}.tar.gz"
-checksum=e2fe7b7afbc07e930491c00a05200c2682cc0576ca8f6da87245a000257f5924
+checksum=6c90715d254d7a7ec0208b29007b64160dd9fb7df4c4aa7f8ec2c9d23114c719
+
+# Some tests are known to fail since 1.1.0
+# This disables these:
+export GTEST_FILTER='-PipeCaptureTest.ReadEmbeddedNULCharacter:PipeCaptureTest.ReadNULByteInMiddleOfMultiByteUTF8Character:'
```

Comment:
You could add the extra test that fails in CI, if you want.

Reply:
Finally done.